sdks/java/io/amazon-web-services2/src/main/java/org/apache/beam/sdk/io/aws2/options/AwsModule.java

```diff
@@ -298,100 +321,38 @@ public void serialize(
   }
 
   /** A mixin to add Jackson annotations to {@link AttributeMap}. */
-  @JsonSerialize(using = AttributeMapSerializer.class)
-  @JsonDeserialize(using = AttributeMapDeserializer.class)
-  private static class AttributeMapMixin {}
-
-  private static class AttributeMapDeserializer extends JsonDeserializer<AttributeMap> {
```

Comment:
Meant to expose http configuration parameters for AWS clients in [AwsOptions](https://github.com/apache/beam/pull/16760/files#diff-d4c58a1b3d2076e891deb2d43272a1383cb0c83494b26e059a1eb9fc9d0dad52L137-L141), but wasn't used at all.
`AttributeMap` is part of the protected internal API of the SDK and shouldn't be used in Beam. It's replaced by 
[HttpClientConfiguration](https://github.com/apache/beam/pull/16760/files#diff-3d87e290202dcedf12176f2c1441cf579feff8367af5478f58a2903bec78ca7dR35)